ui/src/components/navigation_app_bar.jsx

```diff
@@ -46,6 +47,8 @@ export default React.createClass({
   render() {
     const demoCandidateEmail = 'kevin.robinson.0@gmail.com';
     const {userProfile, doLogout} = this.context.auth;
+    const {iconElementRight} = this.props;
```

Comment:
This was to allow callers to set the right-side menu based on whatever the application context is (initially for use within a message popup game) but this isn't used yet.